File: JSBASIC/184_separate_even_odd.js
// 184	184_separate_even_odd.js	Separate even and odd elements of an array into two different arrays.

let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
let evens = [], odds = [];
for (let i = 0; i < arr.length; i++) {
    if (arr[i] % 2 === 0) evens.push(arr[i]);
    else odds.push(arr[i]);
}
console.log("Original:", arr);
console.log("Even numbers:", evens);
console.log("Odd numbers:", odds);

File: JavaBasic/184_separate_even_odd.java
// 184	184_separate_even_odd.java	Separate even and odd elements of an array into two different arrays.
import java.util.*;
public class _184_separate_even_odd {
    public static void main(String[] args) {
        int[] arr={1,2,3,4,5,6,7,8,9,10};
        List<Integer> evens=new ArrayList<>(), odds=new ArrayList<>();
        for (int x:arr) if(x%2==0) evens.add(x); else odds.add(x);
        System.out.println("Evens: " + evens);
        System.out.println("Odds: " + odds);
    }
}
